src/EvoSC.Common/Interfaces/Services/IMatchSettingsService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using EvoSC.Common.Interfaces.Util;
```

Comment:
A suggestion to all these comments is to also add the "exceptions" tag. This gives the implementer of this interface a good idea of what exceptions they have to handle based on the comments provided by us.